packages/next-swr-endpoints/tsup.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from "tsup";
+
+const isProduction = process.env.NODE_ENV === "production";
+
+export default defineConfig({
+  clean: true,
+  dts: true,
+  entry: [
+    "src/index.ts",
+    "src/swr-client-endpoint-loader.ts",
+    "src/swr-server-endpoint-loader.ts",
+  ],
+  format: ["cjs"],
+  minify: isProduction,
+  sourcemap: true,
+  target: "node16",
```

Comment:
There's a few properties both open to change & also adding some additional ones.
EG: Adding `esm` or `iife` formats or defining a `platform` option